Gemfile.lock

```diff
@@ -218,7 +218,6 @@ GEM
       mini_portile2 (~> 2.3.0)
     nokogumbo (1.5.0)
       nokogiri
-    oauth (0.5.4)
```

Comment:
Hmm I think this needs to be reversed to not affect oath work?

Reply:
Woah!! Don't know where it came from!! I'll remove gemfile.lock related changes.

Reply:
For triggering, can we give a button or something which would be visible only to admins and moderators?